### PR TITLE
fix(config): add metadata to Shenzhen Heiman HS1SA-Z

### PR DESCRIPTION
**Pull request by alfakenzo**
Updated some Infos

PR description here

### CODE REVIEW

File: packages/config/config/devices/0x0260/hs1sa-z.json [hunk 1]
```diff
@@ -12,5 +12,11 @@
 	"firmwareVersion": {
 		"min": "0.0",
 		"max": "255.255"
+	},
+	"metadata": {
+		"overwiew": "The smart smoke sensor detects smoke, and sends an alarm signals to alert of danger. When the smart smoke sensor detects smoke, it will send an alarm notification to your controller through wireless Z-Wave communication protocol.",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0260/hs1sa-z.json]: See https://zwave-js.github.io/node-zwave-js/#/config-files/file-format?id=metadata, we have only a limited set of fields at the moment. Overview is not one of them.

File: packages/config/config/devices/0x0260/hs1sa-z.json [hunk 2]
```diff
@@ -12,5 +12,11 @@
...
+		"inclusion": "Device ADD for Inclusion\n- Press the pin 3 times within 1.5s, Green LED is Blinking 3 times within 1 second. If Inclusion Process is successful, Green led will turn off.",
+		"exclusion": "Device Remove for Exclusion\n- Press the pin 3 times within 1.5s. If Exclusion Process is successful, Green led is Blinking 6 times, then turn off.",
+		"wakeup": "Wake up Information\n- Wake up Notification is transmitted every 24 hours by default. Wake up Notification is transmitted after Notification Report is Transmitted."
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0260/hs1sa-z.json]: `wakeup` should indicate how to wake the device up manually, not describe what this does.

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Comment by blhoward2** [comment]
Since this has been abandoned, I've just deleted the unsupported fields so it can be merged.

**Review by AlCalzone — DISMISSED** [review]

**Comment by AlCalzone** [comment]
@zwave-js-bot fix lint

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**